rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,183 @@
+  * technical aspect - The portions of  pull request or commit in php-src or commit to php-src master and dev branches that would be technical changes.
+  * technical decision - Decision on a technical aspect.
+  * TC - PHP Technical Committee.
+
+To further clarify some of the defined terms, following subsection provides some useful examples.
+
+=== Changes ===
+
+It is important to clarify difference between user facing change and technical changes. As it is defined, the technical
+change is any php-src change that is not a user facing change. However, the implementation of user facing change is
+still a technical change. The meaning of that is that the part of the user facing change that defines the user
+interface is not a technical change.
+
+For example if there is a proposal to add accessors for PHP classes, then everything that is visible to user space
+like the constructs and their usage in PHP code is a user facing change. The actual C implementation and tests are
+a technical change.
+
+==== TC membership ====
+
+The TC consists of five elected members.  Any core developer is eligible to run for the TC.  One member of the TC,
+selected by the TC members themselves, is the TC Secretary.
+
+The TC Secretary is responsible for:
+
+  * Administering the election of the next set of TC members.
+  * Documenting and communicating any decisions of the TC.
+  * In case of a tie, the Secretary's vote will be considered the winning position.
+
+=== Elections ===
+
+The election process takes place annually, concurrent with the election of the Release Managers in approximately
+March/April.  (The exact date may vary year to year.)  The sitting TC Secretary officially begins the process with
+a call for nominations.  The nomination period lasts for one week.
+
+During that week, any eligible core developer may self-nominate by posting to the Internals mailing list.  If the
+candidate is paid to work on php-src or PHP extensions, they must state in their nomination which company or organization
+they work for.  Failure to do so will result in disqualification, as determined by the TC Secretary.
+
+Once nominations are over, the voting process will begin.  Eligible voters are the same as for any RFC.
+
+If there are five or fewer nominees, there is no vote and all nominated individuals are elected.
+
+If there are six or more nominees, a two week vote using the Single Transferable Vote method will take place.  The five
+winning candidates at the end of the vote are elected.
+
+The newly elected TC members will elect from their own number a TC Secretary by simple majority.
+
+=== Vacancies ===
+
+Any TC member may resign at any time via an email to the Internals mailing list.  If the resigning member is the TC
+Secretary, then the TC will elect a new Secretary from its number.
+
+The TC may declare one of its members AWOL after no communication from that individual for one month, by a majority vote.
```

Comment:
```suggestion
Should a TC member become inactive for more than 30 days, the TC may declare that member's seat vacant, by a majority vote.
```

This is closer to the wording used by FIG for vacancies.  We don't actually have a term for "this person is missing", just say "become inactive".  I figure following the same language style makes sense.